tangelo/algorithms/classical/tests/test_fci_solver.py

```diff
@@ -81,6 +84,30 @@ def test_fci_H4_interior_frozen_orbitals(self):
         one_rdm, two_rdm = solver.get_rdm()
         self.assertAlmostEqual(energy, mol_H4_sto3g_freeze3.energy_from_rdms(one_rdm, two_rdm), places=5)
 
+    def test_fci_H4_modified_mo_coeff(self):
+        """ Test FCISolver against result from reference implementation with interior frozen orbitals by manually defining
+        a swap operation.
+        """
+
+        mol = SecondQuantizedMolecule(xyz_H4, frozen_orbitals=[1, 3, 4])
+
+        unitary_mat = np.array([[ 0.99799630, -0.05866044,  0.02358562,  0.00246221],
+                                [ 0.05673690,  0.99556613,  0.07416274,  0.01135272],
+                                [-0.02767097, -0.07212301,  0.99605136, -0.04375241],
+                                [-0.00431649, -0.01432819,  0.04272342,  0.99897486]])
+        # Modify orbitals
+        mol.mo_coeff = unitary_mat.T@mol.mo_coeff
+
+        from openfermion import get_sparse_operator
```

Comment:
why did you choose to import these things inside the test and not at the beginning of the module ?

Reply:
I approved, and as soon as this conversation is resolved (whether accompanied with changes or not) we can merge. Cheers

Reply:
I originally was not going to keep this test but realized later that it was good to keep as it confirms that the eigenvalue of FCI and from the FermionOperator are equivalent.